### PR TITLE
Version: 3.33.2

### PR DESCRIPTION
**Pull request by kira-syslogng**
Tarball is available at: https://github.com/syslog-ng/syslog-ng/actions/runs/1044602465

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,135 +1,15 @@
...
-   The default value is 0.01, which operates well with most disk-buffers.
-
-   If the possible size reduction of the truncation does not reach `truncate-size-ratio()` x `disk-buf-size()`,
-   we do not truncate the disk-buffer.
-
-   To completely turn off truncating (maximal disk space loss, maximal IO time saved) set `truncate-size-ratio(1)`,
-   or to mimic the old behavior (minimal disk space loss, minimal IO time saved) set `truncate-size-ratio(0)`.
-   ([#3689](https://github.com/syslog-ng/syslog-ng/pull/3689))
-
+<Fill this block manually from the blocks below>
```
Comment (gaborznagy) [inline NEWS.md:6]: Should we put the disk-buffer bugfix in highlights too?

File: NEWS.md [hunk 2]
```diff
@@ -143,6 +23,6 @@ of syslog-ng, contribute.
 We would like to thank the following people for their contribution:
 
 Andras Mitzki, Antal Nemes, Attila Szakacs, Balazs Scheidler,
-Balázs Barkó, Benedek Cserhati, Gabor Nagy, L4rS6, Laszlo Budai, Laszlo Szemere,
-LittleFish33, László Várady, Norbert Takacs, Peter Czanik, Peter Kokai,
-Todd C. Miller, Tomáš Mózes, Zoltan Pallagi
+Gabor Nagy, Josef Schlehofer, Laszlo Budai, Laszlo Szemere,
```
Comment (gaborznagy) [inline NEWS.md:26]: Can we add  Benedek Cserhati, Balázs Barkó and Szilárd Parrag manually to the news file for now, please?
The script is no up-to-date.
I`ve created a PR for that #3737.

File: NEWS.md (hunk 1 above)
Comment (gaborznagy) [reply NEWS.md:6]: IMHO, it's fine without Highlights, see: https://github.com/syslog-ng/syslog-ng/releases/tag/syslog-ng-3.31.2

**Comment by gaborznagy** [comment]
Besides one comment, approve.

File: NEWS.md (hunk 1 above)
Comment (MrAnno) [reply NEWS.md:6]: I think we should just remove the highlight section. We fixed a disk-buffer regression, that's not a bright news in my opinion.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md (hunk 2 above)
Comment (szemere) [reply NEWS.md]: Thank You for the comment and the related PR! I fixed the list (and made sure, Josef Schlehofer will remain in place).

I don't want to resolve the conversation by myself. Could you please close it, as a double check?

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md (hunk 2 above)
Comment (gaborznagy) [reply NEWS.md]: Checked, it's perfect. Thanks!

**Review by gaborznagy — APPROVED** [review]

**Review by alltilla — APPROVED** [review]

**Pull request merged**